src/oncall/ui/static/js/oncall.js

```diff
@@ -398,10 +411,25 @@ var oncall = {
       router.updatePageLinks();
     },
     renderUserDetails: function(data){
-      var $el = this.data.$page.find(this.data.userDetails);
-      $el.find('.headshot-large').attr('src', data.photo_url).removeClass('placeholder');
-      $el.find('.user-details-name').text(data.full_name);
-      $el.find('.user-details-email').html('<a href="mailto:' + data.contacts.email + '">' + data.contacts.email + '</a>');
+
+      //check if data.photo_url is null , replace it with blank headshot instead
+      if(data.photo_url==null)
```

Comment:
I like this change, but we can simplify the logic a bit in this function.
At the top, we can set the photo_url :
`data.photo_url = data.photo_url || '/static/images/headshot-blank.jpg';`
and then keep the logic below the same as it was, instead of using the if/else and copying the extra lines of logic.

Also, seems like this PR has a conflict with @dwang159's pinned team changes, so it's hard to tell if this block of changes were the only ones made by you, but if this block is it, looks good to me except for the above note.